api/test/unit/utils/test_image_proxy.py

```diff
@@ -219,30 +265,33 @@ def test_get_successful_forward_query_params(mock_image_data):
         .mock
     )
 
-    url_with_params = f"{TEST_IMAGE_URL}?{params}"
+    url_with_params = f"{image_url}?{params}"
 
-    res = photon_get(url_with_params, TEST_MEDIA_IDENTIFIER)
+    res = await photon_get(url_with_params, TEST_MEDIA_IDENTIFIER)
 
     assert res.content == MOCK_BODY.encode()
     assert res.status_code == 200
     assert mock_get.matched
 
 
 @pytest.fixture
-def setup_requests_get_exception(monkeypatch):
+def setup_http_get_exception(monkeypatch):
     def do(exc):
-        def raise_exc(*args, **kwargs):
-            raise exc
+        mock_session = MagicMock(spec=aiohttp.ClientSession)
+        mock_session.get.side_effect = exc
 
-        monkeypatch.setattr("requests.get", raise_exc)
+        monkeypatch.setattr(
+            "api.utils.image_proxy.get_aiohttp_session", lambda: mock_session
+        )
```

Comment:
We don't use `requests` anymore for this, so we need to mock out the aiohttp session instead.